person.rb

```diff
@@ -4,7 +4,7 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
```

Comment:
- [ ] Kindly make sure that rentals in initialized in the initialize method. 
`Hint: @rentals = [ ]`

Reply:
done